bitnami/grafana/templates/deployment.yaml

```diff
@@ -69,6 +69,8 @@ spec:
                   name: {{ include "grafana.adminSecretName" . }}
                   key: {{ include "grafana.adminSecretPasswordKey" . }}
             {{- if .Values.smtp.enabled }}
+            - name: GF_SMTP_ENABLED
+              value: true
```

Comment:
This will be interpreted as a boolean, please add quotation marks around the `true`.